### ISSUE TITLE
→ Accessibility: Components | Carousel/Slideshow

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#1728]
## Description

* The carousel should be announced as a carousel
* The user should be able to know how many slides the carousel has
* Add pause/stop controls when the carousel has autoplay

## Tasks

- [x] [Update specs](https://www.figma.com/design/hxJvuIQQ1xf50xIOQFUZbQ/branch/6z69ibhxIhbYMwIqx51aZ1/%F0%9F%94%B8-Carousel-Specs?m=auto&node-id=483-4281&t=V0edAm2V1Jl7MIx0-1)
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Android ticket
- [ ] iOS ticket
- [x] [Web ticket](https://jira.tid.es/browse/WEB-1813)
- [x] [LOC ticket](https://jira.tid.es/browse/LOC-1833)
